pytest_rerunfailures.py

```diff
@@ -175,6 +175,7 @@ def pytest_runtest_protocol(item, nextitem):
         reports = runtestprotocol(item, nextitem=nextitem, log=False)
 
         for report in reports:  # 3 reports: setup, call, teardown
+            report.rerun = item.execution_count
```

Comment:
To restore the old behaviour, I think we need to subtract from the `execution_count`.
```suggestion
            report.rerun = item.execution_count -1
```